book/tables/ADA/adat01.qmd

```diff
@@ -241,8 +240,7 @@ main_footer(result) <- "ADA = Anti-Drug Antibodies (is also referred to as ATA,
 result
 ```
 
-`r webr_code_labels <- c("setup", "variant1")`
-{{< include ../../_utils/webr.qmd >}}
+`r webr_code_labels <- c("setup", "variant1")` {{< include ../../_utils/webr.qmd >}}
```

Comment:
it looks like the break line matters here
I rendered this as is and I the UI got messed up
`quarto render tables/ADA/adat01.qmd --cache-refresh`
![image](https://github.com/insightsengineering/tlg-catalog/assets/12943682/ba55baf2-7908-4a3a-83a2-8a98e3f6e0ce)


I introduced a new break line you removed and everything is back to normal. Do you have the same?